app/javascript/styles/mastodon/variables.scss

```diff
@@ -27,6 +27,7 @@ $ui-base-lighter-color: lighten($ui-base-color, 26%) !default; // Lighter darkes
 $ui-primary-color: $classic-primary-color !default;            // Lighter
 $ui-secondary-color: $classic-secondary-color !default;        // Lightest
 $ui-highlight-color: $classic-highlight-color !default;        // Vibrant
+$ui-card-secondary-color: $ui-primary-color !default;
```

Comment:
Can we add a comment to the end of this line explaining the purpose of the variable?

Reply:
Hey Ash,

I will absolutely do all of these things. Do you want the descriptions by commit ID? Also,to go along with adding a comment to the new variable, I think I'll disambiguate the name. It's primarily for text on the default "simple background color" which is #FFFFFF.

Reply:
Requested changes made. Please let me know if there's more that needs doing!